### ISSUE TITLE
Overlapping Services

### ISSUE THREAD

**Issue by trackmastersteve** [tom472/mediabox#141]
There are so many containers doing similar tasks. Would it be viable to just add choices in the install script? That way there are only single containers for each task.

**Comment by tom472** [comment]
Yeah I have been thinking about just going with the ARR stack items and pulling the former / other images.
OR as you suggestion come up with a type of choices process in the set-up script.
